yjit/src/asm/x86_64/tests.rs

```diff
@@ -411,3 +411,20 @@ fn basic_capstone_usage() -> std::result::Result<(), capstone::Error> {
         )),
     }
 }
+
+#[test]
+#[cfg(feature = "asm_comments")]
+fn block_comments() {
+    let mut cb = super::CodeBlock::new_dummy(4096);
+
+    cb.add_comment("Beginning".to_string());
+    xor(&mut cb, EAX, EAX); // 2 bytes long
+    cb.add_comment("Two bytes in".to_string());
+    cb.add_comment("Two bytes in".to_string()); // Duplicate, should be ignored
```

Comment:
I think we can avoid the `to_string()` now that the argument is `&str`

Reply:
Good catch. I've switched it back and run the tests with current setup. We have no .to_string() for add_comment(), but we still have it for the slice of the vector that is returned (since we copy them to avoid lifetime issues.)